### DISCUSSION TITLE
Github Copilot: No Suggestions Displayed

### DISCUSSION THREAD

**Discussion by nxmxgoldxx** [community/community#14117] [Copilot Conversations] [answered]
Hello Everyone, 

We just recently installed GitHub Copilot and am not seeing any suggestions displayed upon following the get started instruction markup. Is there something we may be missing in the installation via Visual Studio Code or some settings that need to be configured to prevent the feature from being blocked by another extension, or would this potentially be an issue with a specific /.JSON file setting ? 

Thank you for your support.

**Comment by nxmxgoldxx** [comment, marked as answer]
[SOLVED]

**Reply by mohammedadelabu** [reply]
How did you solve the issue, @nxmxgoldxx as I'm also facing the same issue?

**Reply by nxmxgoldxx** [reply]
Take a look at the bottom tool bar in VSC, there is an indication of what you should expect to see in the image on the getting started page, alas you should see a GitHub icon, but if you do not and see a loading icon, wait for that process to complete its cycle and reload your software it should be implemented. You may need yo click the icon and accept the connection it's attempting to make as well. Let us know, GL

**Comment by mohammedadelabu** [comment]
Thank you, Calente.

It is working on this PC, but I have another laptop using the same github account but the copilot isn't working on it.

So, my question is that does Co-pilot only work on one PC per one github account? If no, please how can I also make it work on my second laptop?

Thank you.
________________________________

**Reply by davecheney** [reply]
Copilot is not restricted to one host per account. 
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by mohammedadelabu** [comment]
Yes, I've chosen GitHub Copilot from the pulldown in Output but it's still not working on the second laptop. I've even signed out and signed in and it's still the same thing.
________________________________

**Reply by davecheney** [reply]
Are you using VSCode, or VSCodium / OSSCode?

**Comment by mohammedadelabu** [comment]
I'm using VS code
________________________________

**Reply by davecheney** [reply]
Please double check, sometimes it can be difficult to spot when an operation system distro is supplying VSCodium.

**Comment by mohammedadelabu** [comment]
Can I uninstall the IDE, though it's VScode and install another one?
________________________________

**Reply by davecheney** [reply]
Generally the problem when copilot does not show up is because the version of vscode is too old, or one of the _from source_ folks like VSCodium or OSSCode.

**Reply by EH-85** [reply]
[Solved] 

It took a few re-installs and logging on/off but I got it sorted and I can now toggle on new projects in Python. 

Can you toggle between the suggestions natively with alt + [ or ] on the Linux build yet with python? 

I cannot, and this is the issue I'm having right now on the latest build and a 60 day free trial--my guess is it is limiting the features because if I open the prompt on a secondary window all the suggestions populate on that. 

Let me  post some screen caps to help illustrate the issue and maybe you can give some clarity on what I'm doing wrong, thanks! 

Toggling next is disabled: https://imgur.com/zgiBnVW
Only 1 suggestion appears until I hit open github pilot: https://imgur.com/cEtBnjh
All the suggestions populate on new window: https://imgur.com/95EJMvx

Sidenote: This is an awesome project for those of us coming back to coding after a haitus.

**Comment by SerhiiVynohradov** [comment]
Updating Github Copilot helped (for NVim + Vundle I did :BundleUpdate).

**Comment by Ant3ney** [comment]
I managed to solve the issue by uninstalling and then reinstalling the extension. After reinstalling the extension, I saw a few prompts appear in vs code and followed them. After that, the suggestions started to appear again.
 
 You may have missed the prompts that appear only when you first install the extension like how I had.

**Reply by wadefleming-nz** [reply]
reinstalling worked for me too

**Comment by mohammedadelabu** [comment]
Thanks to y'all. The GitHub Copilot is working very well now.

Regards.
________________________________
